LICENSE-3rdparty.csv

```diff
@@ -12,10 +12,18 @@ import,com.squareup.okhttp3,Apache-2.0,"Copyright 2019 Square, Inc"
 import,com.squareup.okio,Apache-2.0,"Copyright 2013 Square, Inc"
 import,org.jetbrains,Apache-2.0,Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors
 import,org.jetbrains.kotlin,Apache-2.0,Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors
+import,io.opentracing,Apache-2.0,Copyright 2016-2017 The OpenTracing Authors
```

Comment:
Not necessary to change that right now, but I tried to keep the list sorted on the first two columns. That helps keeping "related" dependencies close together, but you don't have to change that now.

Reply:
Done